From 4ac2915366981cc1264381720a83a34d91e313ed Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Tue, 31 Jan 2017 22:24:52 +0800
Subject: [PATCH 077/249] cm: Remove external/google

We no longer have any dependencies on the SDKs provided there.

Change-Id: I7844f572b71f952c24c67f22eb761ce9c84f7e6e
---
 snippets/cm.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/cm.xml b/snippets/cm.xml
index 04da95a..04bcb3a 100644
--- a/snippets/cm.xml
+++ b/snippets/cm.xml
@@ -11,7 +11,6 @@
   <project path="external/ffmpeg" name="LineageOS/android_external_ffmpeg" />
   <project path="external/fuse" name="LineageOS/android_external_fuse" />
   <project path="external/gello-build" name="LineageOS/android_external_gello_build" />
-  <project path="external/google" name="LineageOS/android_external_google" />
   <project path="external/htop" name="LineageOS/android_external_htop" />
   <project path="external/koush/ion" name="LineageOS/ion" />
   <project path="external/libncurses" name="LineageOS/android_external_libncurses" />
-- 
2.17.1

